File: examples/discover.py
import asyncio
from atlas_index.core.index import AtlasIndex

async def main():
    index = AtlasIndex(facilitator_url='https://facilitator.payai.network')
    
    services = await index.discover(category='AI', network='base')
    
    print(f"Found {len(services)} services")

if __name__ == '__main__':
    asyncio.run(main())

File: examples/discover.ts
import { AtlasIndex } from './core/index';

async function main() {
  const index = new AtlasIndex({
    facilitatorUrl: 'https://facilitator.payai.network',
  });

  const services = await index.discover({
    category: 'AI',
    network: 'base',
  });

  console.log(`Found ${services.length} services`);
}

main().catch(console.error);


